dialogue-apache-hc4-client/src/main/java/com/palantir/dialogue/hc4/ApacheHttpClientChannels.java

```diff
@@ -145,21 +147,40 @@ private static MetricName clientPoolSizeMetricName(String clientName, String sta
 
     /** Intentionally opaque wrapper type - we don't want people using the inner Apache client directly. */
     public static final class CloseableClient implements Closeable {
+        private final String name;
         private final CloseableHttpClient client;
+        private final PoolingHttpClientConnectionManager pool;
         private final ResponseLeakDetector leakDetector;
 
         @Nullable
         private final ExecutorService executor;
 
         CloseableClient(
-                CloseableHttpClient client, ResponseLeakDetector leakDetector, @Nullable ExecutorService executor) {
+                String name,
+                CloseableHttpClient client,
+                PoolingHttpClientConnectionManager pool,
+                ResponseLeakDetector leakDetector,
+                @Nullable ExecutorService executor) {
+            log.info("Apache client created", SafeArg.of("name", name));
```

Comment:
maybe I should actually include the whole state of the builder here - ClientConfiguration, executor etc etc as unsafe args??